doc/source/authors.rst

```diff
@@ -39,7 +39,12 @@ and may not be the current affiliation of a contributor.
 * Chadwick Boulay [16]
 * Björn Müller [13]
 * William Hart [17]
-
+* erikli(github)
+* Jeffrey Gill [18]
```

Comment:
👍

Reply:
We are doing a hackton in Palermo and are about to realase 0.7!!
Thank for your patches.